backend/src/test/java/org/eclipse/tractusx/semantics/registry/AssetAdministrationShellApiSecurityTest.java

```diff
@@ -793,79 +846,19 @@ public void testFindExternalShellIdsBySpecificAssetIdsWithTenantBasedVisibilityE
                     .andExpect(jsonPath("$", hasSize(0)));
 
             // Test lookup with one assetId for tenant two and one without tenantId
-            ArrayNode specificAssetIdsTenantTwo = emptyArrayNode()
-                    .add(specificAssetId(keyPrefix + "findExternalShellIdQueryKey_2", "value_2"))
-                    .add(specificAssetId(keyPrefix + "findExternalShellIdQueryKey_2_1", "value_2_1"));
 
             mvc.perform(
                             MockMvcRequestBuilders
                                     .get(LOOKUP_SHELL_BASE_PATH)
-                                    .queryParam("assetIds", toJson(specificAssetIdsTenantTwo))
+                                    .queryParam("assetIds", mapper.writeValueAsString(List.of(sa1,sa2)))
                                     .accept(MediaType.APPLICATION_JSON)
                                     .with(jwtTokenFactory.tenantTwo().allRoles())
                     )
                     .andDo(MockMvcResultHandlers.print())
                     .andExpect(status().isOk())
                     .andExpect(jsonPath("$", hasSize(1)))
                     // ensure that only three results match
-                    .andExpect(jsonPath("$", contains(getId(firstShellPayload))));
-        }
-
-        @Test
-        public void testFindExternalShellIdsBySpecificAssetIdsWithAnyMatchExpectSuccess() throws Exception {
```

Comment:
Is there any other test which covers this case? 
Why do you remove this test?

Reply:
This is part of Query API and it's not present in this API 3.0 version.